src/server.c

```diff
@@ -2575,7 +2575,7 @@ void resetServerStats(void) {
 }
 
 /* Make the thread killable at any time, so that kill threads functions
- * can work reliably (default cancelability type is PTHREAD_CANCEL_DEFERRED).
+ * can work reliably (default cancellability type is PTHREAD_CANCEL_DEFERRED).
```

Comment:
https://en.wiktionary.org/wiki/cancelability and the note at the bottom of https://www.merriam-webster.com/dictionary/cancel suggests this was an acceptable alternative , but given it only appears once, and it looks like it will get flagged by multiple spellcheckers, it seems wise to just switch to using the spelling commonly used.